.github/workflows/ci.yml

```diff
@@ -0,0 +1,76 @@
+name: CI
+on: push
+
+jobs:
+  ci:
+    name: CI
+    runs-on: ubuntu-latest
+
+    # See https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
+    permissions:
+      # required by aws-actions/configure-aws-credentials
+      id-token: write
+      contents: read
+
+    steps:
+      - uses: actions/checkout@v3
+
+      # Setup AWS credentials to enable uploading to S3 for Riff-Raff.
+      # See https://github.com/aws-actions/configure-aws-credentials
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+          aws-region: eu-west-1
+
+      - uses: actions/setup-node@v3
+        with:
+          cache: yarn
+          cache-dependency-path: app/yarn.lock
+          node-version-file: app/.nvmrc
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+        working-directory: app
+
+      - name: Jest
+        run: yarn jest --coverage
+        working-directory: app
+
+      - name: Type check
+        run: yarn type-check
+        working-directory: app
+
+      - name: Lint
+        run: yarn lint
+        working-directory: app
+
+      # Seed the build number with final build from TeamCity.
+      # (This is used by webpack in the next step)
+
+      # TODO: Update last TeamCity build number before merging!
+
+      - name: Update build number
+        run: |
+          LAST_TEAMCITY_BUILD=6063
+          echo RIFF_RAFF_BUILD=$(( $GITHUB_RUN_NUMBER + $LAST_TEAMCITY_BUILD )) >> $GITHUB_ENV
+
+      - name: Bundle application
+        run: yarn bundle
+        working-directory: app
+        env:
+          WEBPACK_BUILD: ${{ env.RIFF_RAFF_BUILD }}
+
+      - name: Compress for upload
+        run: |
+          cd dist
+          zip -r ../manage-frontend.zip .
+        working-directory: app
+
+      - name: Upload to Riff-Raff
+        uses: guardian/actions-riff-raff@v1
+        with:
+          dryRun: true
```

Comment:
(I imagine this will be set to `false`/removed on merge.)

Reply:
I've left this in whilst TeamCity is still in place so we don't accidentally upload a build and mess up the build numbers, but there's a note on the PR to remove this before merging. (As well as updating `LAST_TEAMCITY_BUILD`.)